index.html

```diff
@@ -46,27 +53,20 @@ <h1 id="title">Survey Form</h1>
         <label for="check" class="input-t">Kotlin</label>
       </div>
       <div class="textarea">
-        <textarea name="review" id="textarea" cols="30" rows="10" placeholder="Comments"></textarea>
+        <textarea
+          name="review"
+          id="textarea"
```

Comment:
- [ ] [REQUIRED] Based on the requirement for this project, you are expected to use the [GitHub inline comments feature](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request) to leave review comments. Please visit the link above to learn how to do that.